src/Cli/dotnet/ParseResultExtensions.cs

```diff
@@ -34,18 +34,31 @@ public static void ShowHelp(this ParseResult parseResult)
             Parser.Instance.Parse(tokenList).Invoke();
         }
 
+        private static string StripPlaceholdersFromString(string input)
+        {
+            // Find and remove all format placeholder strings (and filled runtime strings!) of the form ['"].*['"]
+            // from an input string.
+            // This lets us do fair comparisons against strings that are translated, since the placeholders
+            // may be in various locations, and may have different quote characters in different languages.
+            return System.Text.RegularExpressions.Regex.Replace(input, @"['""](.*)['""]", string.Empty);
```

Comment:
This is greedy .. it will change `Foo "bar" baz "biz" boo`
To `Foo boo` won't it?

You can change to `(.*?)` or `([^'""])*` if you don't want that.

Reply:
Thanks! I'll do so. I want to remove each instance of the pattern.